acceptance/tests/options/config_file/no_external_facts_and_external_dir.rb

```diff
@@ -0,0 +1,34 @@
+# This test verifies that setting external-dir and no-external-facts in the config file
+# results in an error
+test_name "99993: config option no-external-facts conflicts with external-dir" do
+  tag 'risk:low'
+
+  require 'facter/acceptance/user_fact_utils'
+  extend Facter::Acceptance::UserFactUtils
+
+  agents.each do |agent|
+    step "Agent #{agent}: create the exernal-dir"
+    external_dir = agent.tmpdir('external_dir')
+
+    step "Agent #{agent}: create config file"
+    config_dir = agent.tmpdir("config_dir")
+    config_file = File.join(config_dir, "facter.conf")
+    config_content = <<EOM
+global : {
+    external-dir : "#{external_dir}"
+    no-external-facts : true,
+}
+EOM
+    create_remote_file(agent, config_file, config_content)
+
+    teardown do
+      on(agent, "rm -rf '#{external_dir}' '#{config_dir}'")
+    end
+
+    step "Agent #{agent}: config option no-external-facts : true and external-dir should result in an options conflict error" do
+      on(agent, facter("--config '#{config_file}'"), :acceptable_exit_codes => 1) do |facter_output|
+        assert_match(/options conflict/, facter_output.stderr.chomp, "Output does not contain error string")
```

Comment:
nit pick: It is not necessary to call `chomp` on `stderr` when matching via a regex.